pgmpy/factors/CPD.py

```diff
@@ -11,6 +12,7 @@
 from pgmpy.extern import tabulate
 from pgmpy.extern import six
 from pgmpy.extern.six.moves import range, zip
+from warnings import warn
```

Comment:
@anaviltripathi Please start on a fresh branch. And you can [`cherry-pick`](https://git-scm.com/docs/git-cherry-pick) the relevant commits from this branch.